index.js

```diff
@@ -68,7 +106,10 @@ function constructRequest(har, userAgent = false) {
           options.body = encodedParams.toString();
           break;
 
+        case 'multipart/alternative':
         case 'multipart/form-data':
+        case 'multipart/mixed':
+        case 'multipart/related':
```

Comment:
We've supported these three new ones in the [matches-mimetype](https://github.com/readmeio/oas/blob/main/src/lib/matches-mimetype.ts#L21) library in `oas`, and though `oas-to-har` always specifies `multipart/form-data` ([here](https://github.com/readmeio/oas-to-har/blob/main/src/index.js#L293)) we might as well support these other three as this thing should be able to run a HAR from anywhere.

What this doesn't support is if you have an extension on the mime type like `multipart/form-data+json`, and frankly if you're doing something like that you should get blasted into the sun.

Reply:
🚀 🌞